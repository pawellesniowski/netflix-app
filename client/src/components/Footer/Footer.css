.footer {
    background-color: black;
    color: white;
    padding: 10px 30px;
    position: absolute;
    bottom: 0;
    left: 0;
    right: 0;
    text-align: center;
    font-size: 1.5rem;
}


