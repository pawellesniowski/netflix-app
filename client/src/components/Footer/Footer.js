import React from "react";
import "./Footer.scss";

const Footer = () => <footer className="footer" data-cy="footer">Netflix 2019</footer>;

export default Footer;
