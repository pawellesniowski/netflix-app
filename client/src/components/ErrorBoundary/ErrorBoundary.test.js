import React from 'react';
import { shallow } from 'enzyme';

import ErrorBoundary from './ErrorBoundary';
import Footer from '../Footer/Footer';

describe('Error Boundary Component', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<ErrorBoundary><Footer/></ErrorBoundary>);
  });

  test("Snapshot test", () => {
    expect(wrapper).toMatchSnapshot();
  });

  test("Should display error message is displayed when hasError is true", () => {
    wrapper.setState({ hasError: true });
    expect(wrapper.text()).toEqual('Something went wrong')
  });
});
