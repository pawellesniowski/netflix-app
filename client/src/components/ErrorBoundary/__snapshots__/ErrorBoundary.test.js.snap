// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error Boundary Component Snapshot test 1`] = `ShallowWrapper {}`;
